blog/2024-11-27-release-comms-data-link/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+slug: comms-nov2024
+title: |
+  🎉 Highlights from the Latest RenkuLab Releases! 🚀
```

Comment:
Now that I see it in context of our blog, the emojis seem over the top 😉 Also, I'm adding the summary to the title so the post titles are skim-able.
```suggestion
  Highlights from the Latest RenkuLab Releases: Share-able data connectors & Bring-your-own-image to Renku Sessions
```